Pokémon Search App Project

Fikrat2002 opened the pull request:
<img width="953" alt="Screenshot 2024-07-11 153444" src="https://github.com/Fikrat2002/pokemon-search-app/assets/168417613/0395dd37-b08e-4c8d-881a-924ffc1369db">
<img width="586" alt="Screenshot 2024-07-11 230739" src="https://github.com/Fikrat2002/pokemon-search-app/assets/168417613/7442dc21-1536-41ec-8d57-dfa5a628f910">
<img width="498" alt="Screenshot 2024-07-12 110023" src="https://github.com/user-attachments/assets/90f482f8-d49d-4275-8f4a-47acf3a9e5bd">

<img width="456" alt="Screenshot 2024-07-11 211142" src="https://github.com/Fikrat2002/pokemon-search-app/assets/168417613/55fbefa0-2af0-4cf4-9607-8c5f5b51b586">

Fikrat2002 committed:
the html, js and css files created

Fikrat2002 committed:
linters config added

faridafaqiri commented:
Hi @Fikrat2002 🙋‍♀,

Great job! 💯 You've nailed all the project requirements👏Your feature is complete! Congratulations!!! 🎊🏆

Status: need changes❌
Note: A tiny change is required📍
Everything looks perfect. Just add a deployment link in your README file then merge your project 🏆🎊👏

Highlights 🌞
🏆 Successfully implemented a great design
✅ Created a professional repository with your project name
✅ Passed FreeCodeCamp requirements for the project
✅ Used Gitflow correctly by creating a development branch, setting it as default, and making commits on your feature branch
✅ Wrote meaningful commit messages
✅ Set up appropriate linter configurations based on your project's language
✅ No linter errors!
✅ Followed the task and used command-line commits
✅ Submitted a pull request from your feature branch to the development branch and shared the link for review
✅ Provided a clear and detailed description of your work in the PR
✅ Maintained clean code
✅ Created a professional README.md file in your repository
✅ The project is responsive
❌ A function does not work when the name or ID of a Pokémon is mistyped in the input. It should display a message like "Pokémon is not found."

Fikrat2002 committed:
the html, js and css codes applied to this project plus a readme file created

Fikrat2002 committed:
js and css errors fixed

Fikrat2002 committed:
js and css final error fixed

Fikrat2002 committed:
js and css final errors fixed

Fikrat2002 committed:
js spacing error fixed

Fikrat2002 committed:
js final error fixed css updated

Fikrat2002 committed:
readme file updated

Fikrat2002 commented:
> Hi @Fikrat2002 🙋‍♀,
> 
> Great job! 💯 You've nailed all the project requirements👏Your feature is complete! Congratulations!!! 🎊🏆
> 
> Status: need changes❌ Note: A tiny change is required📍 Everything looks perfect. Just add a deployment link in your README file then merge your project 🏆🎊👏
> 
> Highlights 🌞 🏆 Successfully implemented a great design ✅ Created a professional repository with your project name ✅ Passed FreeCodeCamp requirements for the project ✅ Used Gitflow correctly by creating a development branch, setting it as default, and making commits on your feature branch ✅ Wrote meaningful commit messages ✅ Set up appropriate linter configurations based on your project's language ✅ No linter errors! ✅ Followed the task and used command-line commits ✅ Submitted a pull request from your feature branch to the development branch and shared the link for review ✅ Provided a clear and detailed description of your work in the PR ✅ Maintained clean code ✅ Created a professional README.md file in your repository ✅ The project is responsive ❌ A function does not work when the name or ID of a Pokémon is mistyped in the input. It should display a message like "Pokémon is not found."

@faridafaqiri   
I just updated and added this functionality kindly check it out

Fikrat2002 committed:
readme file updated

faridafaqiri commented:
Hi @Fikrat2002 🙋‍♀,

Great job! 💯 You've nailed all the project requirements👏Your feature is complete! Congratulations!!! 🎊🏆

Status: Approved ✅

Highlights 🌞
🏆 Successfully implemented a great design
✅ Created a professional repository with your project name
✅ Passed FreeCodeCamp requirements for the project
✅ Used Gitflow correctly by creating a development branch, setting it as default, and making commits on your feature branch
✅ Wrote meaningful commit messages
✅ Set up appropriate linter configurations based on your project's language
✅ No linter errors!
✅ Followed the task and used command-line commits
✅ Submitted a pull request from your feature branch to the development branch and shared the link for review
✅ Provided a clear and detailed description of your work in the PR
✅ Maintained clean code
✅ Created a professional README.md file in your repository
✅ The project is responsive

Fikrat2002 commented:
> Hi @Fikrat2002 🙋‍♀,
> 
> Great job! 💯 You've nailed all the project requirements👏Your feature is complete! Congratulations!!! 🎊🏆
> 
> Status: Approved ✅
> 
> Highlights 🌞 🏆 Successfully implemented a great design ✅ Created a professional repository with your project name ✅ Passed FreeCodeCamp requirements for the project ✅ Used Gitflow correctly by creating a development branch, setting it as default, and making commits on your feature branch ✅ Wrote meaningful commit messages ✅ Set up appropriate linter configurations based on your project's language ✅ No linter errors! ✅ Followed the task and used command-line commits ✅ Submitted a pull request from your feature branch to the development branch and shared the link for review ✅ Provided a clear and detailed description of your work in the PR ✅ Maintained clean code ✅ Created a professional README.md file in your repository ✅ The project is responsive

@ahzamir my partner approved my project kindly check it once

ahzamir reviewed (approved):
### Hi @Fikrat2002 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
![image](https://github.com/user-attachments/assets/935c7802-596c-4ea0-9708-da857e83d76f)

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

![WinkThumbsUpGIF](https://github.com/ZohalMirkhel/palindrome/assets/96838030/e5ee293e-072a-4806-bde0-b64e13140efa)

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Pokemon Search App project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your partner has reviewed your repo
✔️Your design is responsive

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me @ahzamir in your question so I can receive the notification.**

Fikrat2002 committed:
an html and js functionality added

Fikrat2002 committed:
css style updated

Fikrat2002 committed:
css style updated
